packages/expect/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "jest-matchers",
+  "name": "expect",
```

Comment:
I'd have to go through all the tests.

I'm pretty sure more people use expect than jest-matchers; I'm not sure why the former wouldn't be the one that should survive.